records/2022-Q4-activity-report.md

```diff
@@ -0,0 +1,420 @@
+
+In addition, we have also made some improvements to usability of the site:
+- [docs.scala-lang#2639][docs.scala-lang#2639] to fix the language dropdown of getting-started guide on translated pages.
+- [docs.scala-lang#2637][docs.scala-lang#2637] to ensure all translations have the same layout for the landing page.
+- [docs.scala-lang#2634][docs.scala-lang#2637] make the "outdated page" notice sticky so that the link to redirect to the updated page is always visible.
+
+### Scala Toolkit
+
+For Scala 2 and Scala 3.
+
+The goal of this project is to help the beginner-to-intermediate-level programmers to learn and use Scala in small projects, such as a script.
+To this end, we will publish a series of written tutorials that focus on common programming tasks, such as testing, reading and writing files, sending HTTP requests, etc.
+
+We started writing a few tutorials for unit-testing using the MUnit library, in [scalacenter/docs.scala-lang#7096](https://github.com/sbt/sbt/pull/7096). The Scala Toolkit series will be published later this year.
+
+### An integrated Scala.js ecosystem
+
+For Scala 2 and Scala 3.
+
+The Scala.js ecosystem contains individual pieces of great quality, from the compiler to the UI libraries. However, it is a challenge for every newcomer to find the pieces that are relevant, to connect them, and to build a good development experience. To address this issue, we want to provide a clear “integrated Scala.js ecosystem”. You can find the complete roadmap [here](https://contributors.scala-lang.org/t/the-scala-centers-roadmap-for-a-unified-scala-js-ecosystem/5568).
+
+The second video in the series, [Getting Started with Scala.js, Laminar and ScalablyTyped](https://www.youtube.com/watch?v=hWUAVrNj65c) was published in December 2022.
+We gave talks on the same topic at Scala IO and ScalaCon.
+
+We prepared a written tutorial version of the above, which is currently an open pull request ([scala-js-website#590](https://github.com/scala-js/scala-js-website/pull/590)).
+
+### Extension School
+
+For Scala 3.
+
+Our partnership with the [Extension School](https://extensionschool.ch/learn/effective-programming-in-scala) allows us to provide more support to people learning Scala online. We have been answering the questions of the learners, and providing feedback to their homeworks. In addition, we have added subtitles to all our video lectures.
+
+## Communication
+
+### Let’s Talk About Scala 3 Videos
+
+For Scala 3.
+
+We produced a video about debugging Scala 3 in Metals. This video demonstrates the power of the debugger in a multi-threaded Scala 3 application. We show how to use the debugging steps, the debug console, the watches, and the different types of breakpoints. The video will soon be released on the Youtube platform.
```

Comment:
```suggestion
We produced a video about debugging Scala 3 in Metals. This video demonstrates the power of the debugger in a multi-threaded Scala 3 application. We show how to use the debugging steps, the debug console, the watches, and the different types of breakpoints. The video will soon be released on the Youtube.
```

Platform here is redundant.

Reply:
Should we just say “on YouTube” instead of “on _the_ YouTube”?

Reply:
Ah yes, good catch: `on YouTube`. 👍🏼

Reply:
Done in fce926e36f94706c5bedd331b59ffca522fecc59